policy/lib/main_denies.rego

```diff
@@ -35,14 +37,27 @@ skip_package(package_name) {
 
 # Todo maybe: Skip a rule based on package_name and rule short_name
 
-# Return true if a particular deny rule is effective in the future
+# Return true if a particular rule is effective in the future
 # but not effective right now
 #
-in_future(denial) {
-	# if the denial has effective_on set
-	denial.effective_on
+in_future(rule) {
+	# The rule has effective_on set
+	rule.effective_on
+
+	# The rule is effective in the future but not now
+	time.parse_rfc3339_ns(rule.effective_on) > effective_current_time_ns
+}
+
+# Use the nanosecond epoch defined in the policy config if it is
+# present, otherwise use the real current time
+effective_current_time_ns = now_ns {
+	data.config
```

Comment:
This could be omitted but being explicit here makes it more understandable I think.